internal/inject/slam_service.go

```diff
@@ -53,6 +55,16 @@ func (slamSvcClient *SLAMServiceClient) GetInternalState(ctx context.Context, in
 	return slamSvcClient.GetInternalStateFunc(ctx, in, opts...)
 }
 
+// GetLatestMapInfo calls the injected GetLatestMapInfo or the real version.
+func (slamSvcClient *SLAMServiceClient) GetLatestMapInfo(ctx context.Context, in *v1.GetLatestMapInfoRequest, opts ...grpc.CallOption) (
+	*v1.GetLatestMapInfoResponse, error,
+) {
+	if slamSvcClient.GetLatestMapInfoFunc == nil {
+		return nil, errors.New("no GetLatestMapInfoFunc defined for injected SLAM service client")
+	}
+	return slamSvcClient.GetLatestMapInfoFunc(ctx, in, opts...)
+}
+
 // DoCommand calls the injected DoCommand or the real variant.
```

Comment:
[minor] all the comments in here need to be updated to 
`// XXXX calls the injected XXXXXX if defined else it will return an error`
This is a reminder of the testutils code in rdk which can use real versions.